### PR TITLE
Issue 1 user can create an entry

### PR DESCRIPTION
**Pull request by ChorongShin**

### CODE REVIEW

**Comment by ChorongShin** [comment]
https://user-images.githubusercontent.com/91721272/154177709-41b2a625-b670-4e7a-b670-45868813cac7.mov

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+
+  obj.entryId++;
+
+  data.entries.unshift(obj);
+
+  $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $entryForm.reset();
+
+  obj.entryId = data.nextEntryId;
+  data.nextEntryId++;
```
Comment (thebearingedge) [inline js/main.js:33]: please move these two lines 32 and 33 before the unshift().

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,34 @@
...
+function submitForm(event) {
+  event.preventDefault();
+
+  var obj = {};
+  obj.title = $entryForm.elements.title.value;
+  obj.photo = $entryForm.elements.photo.value;
+  obj.notes = $entryForm.elements.notes.value;
+  obj.entryId = data.nextEntryId;
+
+  obj.entryId++;
```
Comment (thebearingedge) [inline js/main.js:25]: Please remove lines 23-25

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
https://user-images.githubusercontent.com/91721272/154181849-a5405209-63b4-4216-83e2-9e73e2fd7960.mov

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**